### ISSUE TITLE
Land foreclosures + expiration

### ISSUE THREAD

**Issue by codynhat** [Geo-Web-Project/specs#11]
What happens when a land owner fails to pay their taxes? What happens when someone else forecloses on this unpaid land?

**Comment by codynhat** [comment]
After the expiration date is reached, land is automatically put up for sale on a Dutch auction, starting at the self-assessed value. 

@gravenp How long should the auction last? Should it be the same for all parcels?

Intuitively, this makes sense. The land has been for sale for some time at the self-assessed value. The current owner is no longer able to pay taxes and nobody was willing to buy at that price. Slowly lowering the price in a Dutch action should find the optimal market price someone is willing to pay.

After winning an auction, the new self-assessed value is the final purchase price.

**Comment by gravenp** [comment]
The balance in auction length is between giving potential buyers time to discover the auction (and the resulting lower price land) and getting the land back into productive hands sooner rather than later. Because the Harberger system always offers another type of continuous auction, I'd lean towards a shorter timeframe. Let's start with 1 week, consistent across parcels.

**Comment by codynhat** [comment]
Sounds good. And assuming the price decreases linearly? And is set to 0 at the end of the week?

**Comment by gravenp** [comment]
Yeah, keep it simple with linear and 0. There could be justification for some minimum fee (the parcel has been established as being worth something to someone), but I'd favor just getting the parcel into productive use over marginal revenue maximization.

**Comment by codynhat** [comment]
One change: the proceeds from auction go to previous owner, no additional taxes are paid.
